### PR TITLE
update "Connecting to network" page

### PR DESCRIPTION
**Pull request by jmcook1186**
Updates "Connecting to the network" page in "Using Geth" section. Specifically, adds some detail to explanations of the various flags, adds "peer limit" flag, makes tense consistent and minor formatting tweaks.

### CODE REVIEW

File: docs/_interface/peer-to-peer.md [hunk 1]
```diff
@@ -3,72 +3,71 @@ title: Connecting To The Network
...
-To start geth without the discovery protocol, you can use the `--nodiscover` parameter.
-You only want this if you are running a test node or an experimental test network with
-fixed nodes.
 
 ## Checking Connectivity
 
-To check how many peers the client is connected to in the interactive console, the `net`
-module has two attributes that give you info about the number of peers and whether you
-are a listening node.
+The `net` module has two attributes that enable checking node connectivity from the [interactive Javascript console](../_interface/JavaScript-Console.md). These are `net.listening` which reports whether the Geth node is listening for inbound requests,a nd `peerCount` which returns the number of active peers the node is connected to.
```
Comment (holiman) [inline docs/_interface/peer-to-peer.md:57]:
```suggestion
The `net` module has two attributes that enable checking node connectivity from the [interactive Javascript console](../_interface/JavaScript-Console.md). These are `net.listening` which reports whether the Geth node is listening for inbound requests, and `peerCount` which returns the number of active peers the node is connected to.
```

**Review by holiman — APPROVED** [review]

File: docs/_interface/peer-to-peer.md [hunk 2]
```diff
@@ -118,67 +117,72 @@ To check the ports used by geth and also find your enode URI run:
...
-Geth also supports a feature called static nodes if you have certain peers you always want
-to connect to. Static nodes are re-connected on disconnects. You can configure permanent
-static nodes by putting something like the following into
-`<datadir>/geth/static-nodes.json`:
+Geth also supports static nodes. Static nodes are specific peers that are always connected to. Geth reconnects to these peers automatically when it is restarted. Specific nodes are defined to be statc nodes by saving their enode addresses to a json file and providing the file path to Gteh at startup:
+
+```shell
 
-```js
+geth <other flags> datadir/geth/static-nodes.json
```
Comment (s1na) [inline docs/_interface/peer-to-peer.md:129]: hm does this work? I thought this file has to be stored in `<datadir>/geth/static-nodes.json` and then geth will automatically find it without the user having to pass it as argument.

File: docs/_interface/peer-to-peer.md [hunk 3]
```diff
@@ -118,67 +117,72 @@ To check the ports used by geth and also find your enode URI run:
...
-They can be added permanently via a config file `<datadir>/geth/trusted-nodes.json` or temporary via RPC call.
+Geth supports trusted nodes that are always allowed to reconnect, even if the peer limit is reached. They can be added persistently via a config file `<datadir>/geth/trusted-nodes.json` or temporarily using the Javascript console.
+
 The format for the config file is identical to the one used for static nodes.
-Nodes can be added using the `admin.addTrustedPeer()` RPC-call over the js console and removed using the `admin.removeTrustedPeer()` call.
 
-```js
+```shell
+
+geth <other flags> datadir/geth/trusted-nodes.json
```
Comment (s1na) [inline docs/_interface/peer-to-peer.md:168]: Same with this.

File: docs/_interface/peer-to-peer.md [hunk 4]
```diff
@@ -3,72 +3,71 @@ title: Connecting To The Network
...
 
 # Start geth with given config file. Here too the testnet must be specified.
 geth --goerli --config goerli.toml
+
 ```
 
-## How Peers Are Found
+## Finding peers
 
-Geth continuously attempts to connect to other nodes on the network until it has peers. If
```
[the remark this answers is not in the document]
Comment (jmcook1186) [reply docs/_interface/peer-to-peer.md:27]: ah ok. Yes, IIUC the subprotocol negotiation is part of the RLPx session and therefore post-discovery. Thanks!

File: docs/_interface/peer-to-peer.md (hunk 2 above)
Comment (jmcook1186) [reply docs/_interface/peer-to-peer.md:129]: yes you are right. Thanks for the catch - updating now.

**Review by s1na — APPROVED** [review]
LGTM

**Pull request merged**